### DISCUSSION TITLE
How to hook into Towny?

### DISCUSSION THREAD

**Discussion by mindlss** [TownyAdvanced/Towny#4928] [Q&A] [answered]
I tried to depend it in **plugin.yml**, but everythong i get is this error:
![image](https://user-images.githubusercontent.com/68402933/118332193-5c72d380-b512-11eb-8baf-afce6c1b81d2.png)

**Comment by LlmDl** [comment, marked as answer]
https://github.com/TownyAdvanced/Towny/wiki/TownyAPI#getting-a-resident-from-a-player Is probably what you're looking for.

**Reply by mindlss** [reply]
Thanks! exactly what I'm looking for
